project/ScalafixBuild.scala

```diff
@@ -342,4 +342,19 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
       }
     }
   )
+  private def testRulesAgainstPreviousScalaVersions(
+      scalaVersion: String,
+      state: State
+  ): State = {
+    testedPreviousScalaVersions(scalaVersion)
+      .filter(_ != scalaVersion)
```

Comment:
I think this is now redundant (if the `previousVersions` is updated!)
```suggestion
      .filter(_ != scalaVersion)
```